gradle.properties

```diff
@@ -2,6 +2,9 @@
 archives_base_name=wynntils
 enabled_platforms=quilt,fabric,forge
 
+# MixinExtras version
+mixinextras_version=0.2.0-beta.1
```

Comment:
oh, beta. :-/ Are a stable release planned? Have they had a stable 0.1 released?

Reply:
0.1 is out but 0.2 has the cool features

Reply:
Likely soon. It is beta as in author didn't want to make it "release", but released for testing and no apparent issues were reported.

Reply:
Okay, sounds good enough.